content/5-min-overview/index.md

```diff
@@ -35,7 +55,19 @@ called 'trunk' at all.
 
 There are many deciding factors before a development team settles on Trunk-Based Development, but here is a short overview 
 of the practices if they do:
+-->
+
+## 它是什麼？
+
+{{< note title="Notes" >}}
 
+* 在本網站上，提到〝開發者〞一詞時，指的是對相同可建置項目進行自動化測試的〝測試人員〞。
```

Comment:
To clarify, I suggest revising it to: 

```
* 在本網站上，提到〝開發者〞一詞時，也適用於對相同可建置項目進行自動化測試的〝測試人員〞。
```

Reply:
Thank you for your reply. We will discuss this internally. If your suggestion is better, would you authorize us to use it directly?

Reply:
Absolutely, feel free to implement the suggestion.